### DISCUSSION TITLE
Axios not working

### DISCUSSION THREAD

**Discussion by end3r-man** [ionic-team/capacitor#7486] [Q&A] [answered]
axios works on the web after the build on my mobile its not worked at all, tried multiple mobile is their anything have to do before using the package??

**Comment by end3r-man** [comment]
but its working on login page but not working on the register page

**Reply by Kawesom** [reply]
Same issue

**Reply by end3r-man** [reply, marked as answer]
> Same issue

while capacitor http is enabled axios seems to have problem with file upload after disabling the capacitor http is works
